api/api/controllers/test/test_agent_controller.py

```diff
@@ -1184,3 +1184,26 @@ async def test_get_agent_summary_os(mock_exc, mock_dapi, mock_remove, mock_dfunc
     mock_exc.assert_called_once_with(mock_dfunc.return_value)
     mock_remove.assert_called_once_with({})
     assert isinstance(result, ConnexionResponse)
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("mock_request", ["agent_controller"], indirect=True)
+@patch('api.configuration.api_conf')
+@patch('api.controllers.agent_controller.DistributedAPI.distribute_function', return_value=AsyncMock())
+@patch('api.controllers.agent_controller.remove_nones_to_dict')
+@patch('api.controllers.agent_controller.DistributedAPI.__init__', return_value=None)
+@patch('api.controllers.agent_controller.raise_if_exc', return_value=CustomAffectedItems())
+async def test_get_agent_uninstall_permission(mock_exc, mock_dapi, mock_remove, mock_dfunc, mock_exp, mock_request):
+    """Verify 'get_agent_uninstall_permission' endpoint is working as expected."""
+    result = await get_agent_uninstall_permission()
+    mock_dapi.assert_called_once_with(f=agent.agent_check_uninstall_permission,
```

Comment:
```suggestion
    mock_dapi.assert_called_once_with(f=agent.check_uninstall_permission,
```

Reply:
Fixed in efb4a691b97c462325ee2b7e9c4faa616a435692.